docs/_docs/04_reference/config-blocks-and-attributes.md

```diff
@@ -377,19 +377,26 @@ more about the inheritance properties of Terragrunt in the [Filling in remote st
 section](/docs/features/keep-your-remote-state-configuration-dry/#filling-in-remote-state-settings-with-terragrunt) of the
 "Keep your remote state configuration DRY" use case overview.
 
-The `include` block supports the following arguments:
+You can have more than one `include` block, but each one must have a unique label. Note that a bare `include` block with
+no label (`include {}`) is a short hand for an `include` block that uses the label `""` (equivalent to `include "" {}`).
 
+`include` blocks support the following arguments:
+
+- `name` (label): You can define multiple `include` blocks in a single terragrunt config. Each include block
+  must be labeled with a unique name to differentiate it from the other includes. E.g., if you had a block `include
```

Comment:
I'd be tempted to make labeled `include` blocks the default / recommended option and only mention the unlabeled one as a deprecated pattern kept around for backward compat. Again, the goal is to have one, simple, recommended path to follow.